src/_common/content/content-viewer/content-viewer.vue

```diff
@@ -13,4 +13,9 @@
 <style lang="stylus" scoped>
 >>> p > code
 	white-space: normal
+
+// Because white space is rendered out in the editor, we want the viewer to get as close to that
+// as possible. HTML by default collapses white space, and this overrides that behavior.
+>>> p > span
+	white-space: break-spaces
```

Comment:
since this seems to be a little bit of a "new" property, can you use pre-wrap first and then break-spaces next since pre-wrap is the closest to break-spaces for older browsers:
```
white-space: pre-wrap
white-space: break-spaces
```